index.html

```diff
@@ -6,7 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css">
   <script src="data.js"></script>
-  <script src="index.js"></script>
+  <script src="index.js" defer></script>
```

Comment:
good good. you can also handle this by moving your script tag to just above the `</body>` tag in your HTML file! 

AND we're going to learn another way to handle this soon, too!